e2e/fixtures/rsc-router/src/entries.tsx

```diff
@@ -6,6 +6,7 @@ export default unstable_defineRouter(
   // getPathConfig
   async () =>
     STATIC_PATHS.map((path) => ({
+      pattern: `/^${path}/`,
```

Comment:
Is this correct? What's `^`?

Reply:
It is not! It should be `/^${path}$/`!

Reply:
Its a regular expression.

Reply:
But it matches correctly for this case nonetheless.

Reply:
Shouldn't it be this? I don't know why it works.

```suggestion
      pattern: `^${path}$`,
```

Reply:
You are right ...

Reply:
It works because it has no effect in this case. Its just there to satisfy typescript.